internal/rbd/controllerserver.go

```diff
@@ -607,9 +607,9 @@ func (cs *ControllerServer) DeleteVolume(ctx context.Context, req *csi.DeleteVol
 	defer cs.OperationLocks.ReleaseDeleteLock(volumeID)
 
 	rbdVol := &rbdVolume{}
```

Comment:
golang-ci complains about this, you'll need to use
```golang
var rbdVol *rbdVolume
```

Reply:
updated now